test/plugin/scenarios/struts2.5-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,49 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+registryItems:
+  applications:
+    - {struts2.5-scenario: 2}
+  instances:
+    - {struts2.5-scenario: 1}
+  operationNames:
+    - struts2.5-scenario: [/struts2.5-scenario/struts-scenario/case.action]
+  heartbeat: []
+segmentItems:
+  - applicationCode: struts2.5-scenario
+    segmentSize: gt 1
+    segments:
+      - segmentId: not null
+        spans:
+          - operationName: /struts2.5-scenario/struts-scenario/case.action
+            operationId: 0
+            parentSpanId: -1
+            spanId: 0
+            spanLayer: Http
+            startTime: gt 0
+            endTime: gt 0
+            componentId: 15
+            componentName: ''
+            isError: false
+            spanType: Entry
```

Comment:
I prefer to enhance this case, we should check the ref by adding a http call.

Reply:
Done.